tests/installation/isosize.pm

```diff
@@ -11,7 +11,12 @@ sub run {
     if (!$size || !$max || $size > $max) {
         $result = 'fail';
     }
-    bmwqemu::diag("check if actual iso size $size fits $max: $result");
+    if (!defined $size) {
```

Comment:
The thing is: I want to differentiate between size being zero or not defined at all. That is why I use the explicit "if !defined"